src/Sylius/Bundle/ApiBundle/Serializer/ProductVariantNormalizer.php

```diff
@@ -61,10 +61,13 @@ public function normalize($object, $format = null, array $context = [])
         try {
             $data['price'] = $this->priceCalculator->calculate($object, ['channel' => $channel]);
             $data['originalPrice'] = $this->priceCalculator->calculateOriginal($object, ['channel' => $channel]);
-            $data['lowestPriceBeforeDiscount'] = $this->priceCalculator->calculateLowestPriceBeforeDiscount(
-                $object,
-                ['channel' => $channel],
-            );
+
+            if (\method_exists($this->priceCalculator, 'calculateLowestPriceBeforeDiscount')) {
+                $data['lowestPriceBeforeDiscount'] = $this->priceCalculator->calculateLowestPriceBeforeDiscount(
+                    $object,
+                    ['channel' => $channel],
+                );
+            }
         } catch (MissingChannelConfigurationException) {
             unset($data['price'], $data['originalPrice']);
```

Comment:
Not sure why this unset is necessary since either all three are found or not found and only one of the two cannot be set, but to keep things the same I guess it needs third `unset` as well.

```suggestion
            unset($data['price'], $data['originalPrice'], $data['lowestPriceBeforeDiscount']);
```